storybook/config.js

```diff
@@ -1,15 +1,17 @@
 import { addDecorator, configure } from '@storybook/react';
+import StoryRouter from 'storybook-react-router';
 
 import LocaleDecorator from './addons/LocaleDecorator';
-import Row from './addons/Row';
+import SuspenseDecorator from './addons/SuspenseDecorator';
 
 import '../src/app/styles/main.css';
 
+addDecorator(SuspenseDecorator);
 addDecorator(LocaleDecorator);
-addDecorator(Row.Decorator);
+addDecorator(StoryRouter());
```

Comment:
why dont we want the Row anymore?

Reply:
We don't have the row in the common components.

We want to use other grid.

The naming is wrong, it should be just 'offset' or 'space' as we want to have margin arround the components so they are not sticked on the iframe window.